packages/atomic/src/components/result-lists/atomic-result-children/atomic-result-children.tsx

```diff
@@ -155,7 +155,7 @@ export class AtomicResultChildren {
         engine={this.bindings.engine}
         density={this.displayConfig.density}
         imageSize={this.imageSize || this.displayConfig.imageSize}
-        classes="child-result"
+        classes={`child-result ${isLast ? 'last-child' : ''}`.trim()}
```

Comment:
could pure css be used to target & style the last child?

Reply:
> <img alt="Screen Shot 2022-05-13 at 10 13 22 AM" width="330" src="https://user-images.githubusercontent.com/4923043/168302584-e1735ca7-8089-4510-bea3-837ebff85ecb.png">
> 
> is the highlighting working properly? or is it indexed this way
<img width="549" alt="Screenshot 2022-05-13 at 15 59 05" src="https://user-images.githubusercontent.com/30511433/168311347-a567b607-30f3-4c3f-8929-a644396cb8a3.png">

looks like it's the highlighting, right?